.github/workflows/github-pr-linux-container.yaml

```diff
@@ -1,7 +1,6 @@
 name: Hosted Runners (linux, CUDA)
 
 on:
-  push:
```

Comment:
What about
```suggestion
 push:
    branches:
      - develop
```
like we do in https://github.com/kokkos/kokkos/blob/33367fcf08aef9b58ca61cefcc75f2e9b8d95e6b/.github/workflows/continuous-integration-stager.yml#L4-L6 so that we test commits after merging?